### ISSUE TITLE
400 "Bad Request" when limit is not explicitly stated in chef_role

### ISSUE THREAD

**Issue by kevinkarwaski** [gofullstack/capistrano-chef#11]
I ran into an issue when upgrading from capistrano-chef 0.0.2 to 0.0.4 which I wanted to report. Perhaps this is something on my end but I'm not sure. 

It appears if the limit attribute is not explicitly stated in the chef_role, the search query to the chef server fails with a 400 "Bad Request":

```
$ cap -v staging deploy
~/.rvm/rubies/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2632:in `error!': 400 "Bad Request" (Net::HTTPServerException)
from ./config/deploy/staging.rb:5:in `load'
```

This fails:

```
chef_role :web, "chef_environment:my_env_name AND roles:my_role",:attribute => :fqdn
```

This succeeds:

```
chef_role :web, "chef_environment:my_env_name AND roles:my_role",:attribute => :fqdn,:limit => 100
```

I'm not sure why this is happening as it appears limit is assigned a default value of 1000 in:
https://github.com/cramerdev/capistrano-chef/blob/v0.0.4/lib/capistrano/chef.rb

This was tested using:
Chef: 0.10.10
Capistrano: 2.13.4
Capistrano-chef: 0.0.4

Happy to share more info if needed. Thanks!

**Comment by smith** [comment]
Thanks for reporting this. I ran into the same problem and just put `limit: 100` in my deploy.rb to work around it. I'll take a look and hopefully can get a fix.

**Comment by kevinkarwaski** [comment]
Glad to help... I used the same workaround as you. :)

**Comment by smith** [comment]
I think this was fixed in pull #12.
